status-whisper-usage-spec.md

```diff
@@ -245,6 +246,29 @@ Public and group messages are encrypted using symmetric encryption and the key i
 
 One-to-one messages are encrypted using asymmetric encryption.
 
+## Message confirmations
+
+Sending a message is a complex process where many things can go wrong. Message confirmations tells a node that a message originating from it has been received by its peers.
```

Comment:
what does "received by its peers" mean? Which peers? Wouldn't this result in a lot of extra traffic?

Reply:
Node's peers. I think it's pretty clear in this context -- each node has its own peers (other nodes it is connected to).

> Wouldn't this result in a lot of extra traffic?

Yes, it might increase the traffic quite a lot. Currently, only mailservers sent confirmations. If one sends N batches, and there are M nodes sending confirmations, we have N*M additional packets floating around. They are not broadcasted though, only one hop.

Reply:
Ack. Do we have any statistics for how big this overhead is in terms of BW?

Reply:
What is BW? 

No, we don't have any stats. It's not a big deal from a client perspective. For each batch of envelopes, the client receives a single, quite small packet. The only thing which is not necessary is the fact that a node does not know whether a peer wants the confirmation at all. This can be improved.

Reply:
BW is bandwidth